### DISCUSSION TITLE
Expected timeline of Jupyterlab 4 support

### DISCUSSION THREAD

**Discussion by MRYingLEE** [jupyterlite/jupyterlite#1115] [General]
I know there is a roadmap already. Just wonder whether there is an expected timeline for the support.

Thanks,

Ying

**Comment by jtpio** [comment]
@MRYingLEE there is ongoing work in https://github.com/jupyterlite/jupyterlite/pull/1019. Hopefully in a couple of days or so, followed by a `0.2.0` pre-release to get it out.
